indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,3139 @@
+
+### 3.3 Other standards
+
+None
+
+### 3.4 Collinearities with other indicators
+
+Different functional plant community indicators for different environmental variables (light, moisture, ...) might show a certain degree of co-linearity with one another. Therefore, any aggregation of scaled functional plant community indicators to one single functional plant community indicator index should rely on using the indicator with worst scaled value and not on an average of all indicators.
+
+Functional plant community indicators can potentially exhibit co-linearities with other vegetation-oriented indicators, such as NDVI, though this depends greatly on which functional plant community indicator ends up as the leading one (i.e. the one with the lowest scaled score).
+
+### 3.5 Impact factors
+Plant community composition is sensitive to consistent directional changes in environmental drivers, but the main pressures may vary between ecosystems.
+
+Farmlands and semi-natural lowland ecosystems have been kept open by historic human activities and land use. With land-use change, intensification of certain agricultural areas and abandonment of many other, encroachment of shrubs and trees is one of the main challenges today for anthropogenically open systems which change the plant community composition in a directional way. In addition, climatic changes, pollution and wear and tear (for instance from recreational overuse) alter the environmental conditions for both human influenced and naturally open ecosystems in directional ways, for instance through increased precipitation, drought spells and high critical loads of nitrogen deposition. 
+
+Mountain ecosystems are characterized as limited by cold temperatures and usually low availability of nutrients. The main challenge today are thus climate warming and climate-warming related species range shifts (which introduce strong changes in biotic interactions between species) and pollution (e.g. nitrogen deposition) which affect the plant community composition in a directional way.
+
+Wetland ecosystems are primarily characterized by high moisture and are thus threatened by drainage and shrub/tree encroachment, which both change the plant community composition dramatically. In addition, pollution can cause directional shifts in plant communities (e.g. nitrogen deposition).
+
+The plant community composition in forest ecosystems is subject to changes with climate warming and precipitation changes as well as pollution and disturbance from human infrastructure.
+
+Also alien species represent a major threat to all ecosystems, and this pressure may cause shifts in functional plant community signatures as found for naturally open systems around the Oslofjord (Evju et al. 2023). Alien species are covered in a dedicated indicator elsewhere.
+## 4. Reference condition and levels
+
+### 4.1 Reference condition
+
```

Comment:
Add to this section if relevant:
"In Norway, the reference condition for natural ecosystems have been one of largely intact ecosystems without large-scale human modifications (Nybø and Evju (red) 2017). For semi-natural ecosystems, human influence is allowed, and indeed a prerequisite, but then in the sense of appropriate, stable and traditional land-use practices."